solarcity.py

```diff
@@ -587,11 +596,13 @@ def main():
                 special = "high"
             elif production_min is None or production < data['data'][production_min]['production']:
                 special = "low"
-            tweet_production(daylight_hours, cloud_cover, production, special)
-            data['config']['lastdailytweet'] = current_day
+            if not args.no_tweet:
+                tweet_production(daylight_hours, cloud_cover, production, special)
+                data['config']['lastdailytweet'] = current_day
             data_changed = True
-            # Now upload to pvoutput
-            upload_to_pvoutput(data, current_day)
+            if args.pvoutput is not None:
```

Comment:
This changes the behavior. If you provide "--pvoutput" a parameter it ONLY sends output to pvoutput. If you do daily it does the daily stuff and the output to pvoutput. Could re-order the parsing to catch daily first or solve it another way.